packages/webview_flutter/test/webview_flutter_test.dart

```diff
@@ -100,12 +101,129 @@ void main() {
     expect(() => controller.loadUrl('flutter.io'), throwsA(anything));
     expect(platformWebView.lastUrlLoaded, isNull);
   });
+
+  testWidgets('Check can go back', (WidgetTester tester) async {
+    WebViewController controller;
+    await tester.pumpWidget(
+      WebView(
+        onWebViewCreated: (WebViewController webViewController) {
+          controller = webViewController;
+        },
+      ),
+    );
+
+    expect(controller, isNotNull);
+
+    final bool canGoBackAnyPageLoaded = await controller.canGoBack;
+
+    expect(canGoBackAnyPageLoaded, false);
+
+    await controller.loadUrl('https://flutter.io');
```

Comment:
nit: my personal preference is to keep a separate test for each test case, so when the test fails just by the name of the test we can tell which scenario failed.

So e.g I'd have separate tests named : "Can't go back before loading a page", "Can't go back with no history", "Can go back".

(Same comment for the other tests).

Reply:
I separated it. Is it ok? -> 7554d1a